src/SDK/MonitoringApplianceEmulator/Controls/TracesControl.xaml.cs

```diff
@@ -0,0 +1,64 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="TracesControl.xaml.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace Microsoft.Azure.Monitoring.SmartDetectors.MonitoringApplianceEmulator.Controls
+{
+    using System.Collections.Specialized;
+    using System.Linq;
+    using System.Windows.Controls;
+    using System.Windows.Data;
+    using System.Windows.Input;
+    using Microsoft.Azure.Monitoring.SmartDetectors.MonitoringApplianceEmulator.ViewModels;
+    using Unity;
+
+    /// <summary>
+    /// Interaction logic for TracesControl.xaml
+    /// </summary>
+    public partial class TracesControl : UserControl
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TracesControl"/> class.
+        /// </summary>
+        public TracesControl()
+        {
+            this.InitializeComponent();
+            this.DataContext = App.Container?.Resolve<TracesControlViewModel>() ?? new TracesControlViewModel();
+            ((INotifyCollectionChanged)this.TracesGrid.Items).CollectionChanged += (sender, args) =>
```

Comment:
>CollectionChanged [](start = 62, length = 17)

maybe there is an event like "user clicked somewhere in this control" that can stop the auto scrolling

Reply:
Added to US #1381627

---
In reply to: [231514301](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/13#discussion_r231514301) [](ancestors = 231514301)